openwisp_radius/api/serializers.py

```diff
@@ -128,11 +141,20 @@ def validate(self, data):
             raise serializers.ValidationError(
                 {'number_of_users': _('The field number_of_users cannot be empty')}
             )
-        return super().validate(data)
+        validated_data = super().validate(data)
+        # Additional Model Validation
+        batch_data = data.copy()
+        batch_data.pop('number_of_users', None)
+        slug = batch_data.pop('organization_slug')
+        batch_data.update(organization=Organization.objects.get(slug=slug))
+        instance = self.instance or self.Meta.model(**batch_data)
+        instance.full_clean()
```

Comment:
While debugging on the web API interface, I noticed another issue that I didn't think about before, so I have added a model check with some modification. :smile: